legal/Container-Images-Legal-Notice.md

```diff
@@ -1 +1,3 @@
-**Legal Notice:** Microsoft containers are licensed under various licenses depending upon the container’s content. If the container has a Debian, Ubuntu or Alpine Linux base image and you want to retrieve the licenses, copyright notices or source code information for the base image and its official packages, please review the [Linux Legal Metadata](https://aka.ms/mcr/osslinuxmetadata) instructions file. If you are using a Windows based container the governing licenses are referenced in this Docker Hub “License” section or in the container itself, usually as a NOTICE or LICENSE file.  All other components (whether open or closed source) are licensed as indicated either in this Docker Hub “License” section or in the container itself.
\ No newline at end of file
+**Legal Notice:** Microsoft containers are licensed under various licenses depending upon the container’s content. If the container has a Debian, Ubuntu or Alpine Linux base image and you want to retrieve the licenses, copyright notices or source code information for the base image and its official packages, please review the [Linux Legal Metadata](https://aka.ms/mcr/osslinuxmetadata) instructions file. If you are using a Windows based container the governing licenses are referenced in this Docker Hub “License” section or in the container itself, usually as a NOTICE or LICENSE file.  All other components (whether open or closed source) are licensed as indicated either in this Docker Hub “License” section or in the container itself.
```

Comment:
> Microsoft containers

We should be talking about "container images". That's the topic of licensing and the artifact that we can reason about. It's very easy to host additional content in a running container. We make no legal claims on the content hosted within a container.

Reply:
We should bring this with @OSSAttorney and discuss in the meeting he has scheduled - right now we use the PR for technicalities (i.e. content is not published without approval) and not for approval on the content itself.